cli/src/main/java/org/infinispan/cli/commands/rest/Grant.java

```diff
@@ -1,10 +1,11 @@
-package org.infinispan.cli.commands;
+package org.infinispan.cli.commands.rest;
```

Comment:
are these commands (evict, abort, end, revoke, replace etc) to be implemented in the future?

Reply:
They are placeholders I created for features we had in the legacy cli. We do need to add them back

Reply:
I've removed them